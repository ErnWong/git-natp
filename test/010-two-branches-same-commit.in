A--B--C master feature dev release
