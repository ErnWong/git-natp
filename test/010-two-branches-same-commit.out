A
B A
C B
[master] C
[feature] C
[dev] C
[release] C
